src/test/utils.test.ts

```diff
@@ -108,6 +108,14 @@ describe('Utils', () => {
 			yamlDoc.should.deep.equal(yamlDoc2);
 			fs.unlinkSync(fpTest);
 		});
+
+		it('should return undefined for yaml files which are not kind VirtualDatabase', () => {
+			const fpTest: string = path.resolve(__dirname, '../../testFixture', `dummy.yaml`);
+			fs.writeFileSync(fpTest, 'test');
+			const yamlDoc:DataVirtConfig = utils.loadModelFromFile(fpTest);
+			should.not.exist(yamlDoc);
+			fs.unlinkSync(fpTest);
```

Comment:
When test classes will be created/another PR: the clean of the created files should be done in the teardown otherwise when the test is failing the file is not deleted and can impact other tests.

Reply:
You are right.

Reply:
Fixed in latest commit.